### ISSUE TITLE
MAC OS Monterey not supported

### ISSUE THREAD

**Issue by Lionhunt3r** [jwise/HoRNDIS#145]
I get an installation ERROR, when trying to install it. I got no further information...

**Comment by coasterleung** [comment]
Bus:	USB
  Vendor Name:	Xiaomi


  Product Name:	MI 9 Transparent Edition
  Vendor ID:	0x2717
  Product ID:	0xff80
  USB Link Speed:	Up to 480 Mb/s
  Driver:	com.joshuawise.kexts.HoRNDIS
  BSD Device Name:	en10
  MAC Address:	7e:6d:eb:df:87:55
  AVB Support:	No
  Maximum Link Speed:	Unknown

I did remember that I installed failed with brew install command,I can't figure out why it can be used now

**Comment by z88kat** [comment]
Yes, there appears to be a problem with both brew and the package

**Comment by clippycoder** [comment]
Happy to announce it successfully works on Monterey! I have just set it up (and am using it right now). You get an error  because you cannot edit system extensions without disabling SIP first. When installing after disabling SIP, it will still give you the error, but now you can go into system preferences and allow the extension through (click quit on the installer _first_ ). Unfortunately, re-enabling SIP will cause the installation to fail. In my use case, it is worth disabling SIP, but you'll have to decide for yourself.

**Comment by 282855213** [comment]
> 

Could you tell me the more details about the install step,I just disabled the csrutil ,csrutil status
System Integrity Protection status: disabled.
but when I install horndis is fail still , i have no idea how to do, i hope to get your help ,thanks

**Comment by clippycoder** [comment]
How far did you get in the installation? Did you allow the extension through system preferences?

**Comment by zalun** [comment]
Same problem - it doesn't show up in the `Security & Privacy / Allow apps downloaded from` section.
Is there another way for the extension to be allowed?

**Comment by Fan-Xiaohua** [comment]
> 同样的问题 - 它没有出现在该`Security & Privacy / Allow apps downloaded from`部分中。
> 是否有其他方法可以允许扩展？

same，my version is Monterey 12.2.1

**Comment by MuhammadimYusoff** [comment]
I just downloaded the package and got error 

`brew install --cask horndis
==> Caveats
horndis requires a kernel extension to work.
If the installation fails, retry after you enable it in:
  System Preferences → Security & Privacy → General

For more information, refer to vendor documentation or this Apple Technical Note:
  https://developer.apple.com/library/content/technotes/tn2459/_index.html

==> Downloading https://github.com/jwise/HoRNDIS/releases/download/rel9.2/HoRNDIS-9.2.pkg
Already downloaded: /Users/isky/Library/Caches/Homebrew/downloads/eabca1a09c01d4ed6d3818d32f50051a3f3071ceba98f6c8fa9704de0284dfcf--HoRNDIS-9.2.pkg
Error: macOS El Capitan, Sierra, High Sierra or Mojave is required for this software.`

is it really? on the website it says for MacOS 10.11 and above for 9.2 version

anyone know how to fix it?

**Comment by Jayiitb** [comment]
Same thing. Is there any alternative of using android cable connection for internet??
I am using Monterey 12.4.

**Comment by clippycoder** [comment]
@Jayiitb you can use easytether pro. It's very old, but still works even on M1. It cost $10 but it isn't throttled by my ISP.
